ispdb/comcast.net.xml

```diff
@@ -9,46 +9,61 @@
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
```

Comment:
Hey Alex, have you tested how a Thunderbird _without_ the OAuth info hardcoded (and without the ability to use the OAuth config provided here) will react to this? I don't remember whether TB skips OAuth configs, if TB doesn't know the OAuth config, and falls back to password auth as before (that's what it should do), or whether it tries to use OAuth2 and then fails during login (that's how I read the code - this should be fixed first).

Reply:
I don't believe I have a reasonable way to test this?  However, Tbird is not the only consumer of this.  FWIW, I'm also about submit a patch to Tbird to enable/hardcode Oauth for the same set of users.

Reply:
You can try with Thunderbird 102. To use your config file, you can place it on your disk in the TB install directory in file _tb-install-dir_/isp/comcast.net.xml . (Details on https://www.bucksch.org/1/projects/thunderbird/autoconfiguration/ )

Reply:
(https://bugzilla.mozilla.org/show_bug.cgi?id=1844810)

Reply:
I tested it, and unfortunately, it fails :-( I'll post another message on the PR, outside the review here.